dll/cpl/desk/background.c

```diff
@@ -1146,19 +1146,15 @@ static VOID
 OnCustomButton(HWND hwndDlg, PBACKGROUND_DATA pData)
 {
     HPROPSHEETPAGE hpsp[1] = {0};
-    PROPSHEETHEADER psh;
-    PROPSHEETPAGE psp;
+    PROPSHEETHEADER psh = {sizeof(psh)};
+    PROPSHEETPAGE psp = {sizeof(psp)};
 
-    ZeroMemory(&psh, sizeof(psh));
-    psh.dwSize = sizeof(psh);
     psh.dwFlags = PSH_NOAPPLYNOW;
     psh.hwndParent = GetParent(hwndDlg);
     psh.hInstance = hApplet;
     psh.pszCaption = MAKEINTRESOURCE(IDS_DESKTOP_ITEMS);
     psh.phpage = hpsp;
 
-    ZeroMemory(&psp, sizeof(psp));
-    psp.dwSize = sizeof(psp);
```

Comment:
Refactored these two thanks to Mark's comment: https://github.com/reactos/reactos/pull/4766#discussion_r992488804